src/components/performance/views.tsx

```diff
@@ -0,0 +1,200 @@
+                  'number of organisations',
+                  'Number',
+                  numberLabel,
+                  [
+                    { ...props.organizations[0], label: 'billable' },
+                    { ...props.organizations[1], label: 'trial' },
+                  ],
+                )}
+                units="number"
+                metric="mOrganizations"
+                downloadLink={props.linkTo('performance.download', { metric: 'mOrganizations' })}
+              >
+                <div className="govuk-grid-column-one-half">
+                  <p className="govuk-body">
+                    <strong className="govuk-!-font-size-48">{latestValue(props.organizations[0])}</strong><br />
+                    <small className="govuk-!-font-size-27">currently billable organisations</small>
+                  </p>
+                </div>
+                <div className="govuk-grid-column-one-half">
+                  <p className="govuk-body">
+                    <strong className="govuk-!-font-size-48">{latestValue(props.organizations[1])}</strong><br />
+                    <small className="govuk-!-font-size-27">currently trial organisations</small>
+                  </p>
+                </div>
+              </Metric>
+            : <p className="govuk-body">The number of organisations chart is currently not available.</p> }
+
+          {props.applicationCount
+           ? <Metric
+                id="applicationCount"
+                format="number"
+                title="Number of running applications"
+                description={`The number of tenant applications running in all organisations in the ${props.region}
+                region.`}
+                chart={drawLineGraph(
+                  'number of running applications',
+                  'Number',
+                  numberLabel,
+                  props.applicationCount,
+                )}
+                units="number"
+                metric="mApplicationCount"
+                downloadLink={props.linkTo('performance.download', { metric: 'mApplicationCount' })}
+              >
+                <div className="govuk-grid-column-one-half">
+                  <p className="govuk-body">
+                    <strong className="govuk-!-font-size-48">{latestValue(props.applicationCount[0])}</strong><br />
+                    <small className="govuk-!-font-size-27">currently running applications</small>
+                  </p>
+                </div>
+              </Metric>
+            : <p className="govuk-body">The number of running applications chart is currently not available.</p> }
+
+          {props.serviceCount
+            ? <Metric
+                id="serviceCount"
+                format="number"
+                title="Number of backing services"
+                // eslint-disable-next-line max-len
+                description={`Applications rely on backing services such as databases, an email delivery service or a
```

Comment:
Applications rely on backing services such as databases, a caches or a monitoring system. The number of backing services running in the ${props.region} region.